docs/faq/massplaymedia.md

```diff
@@ -34,4 +34,4 @@ There are additional options as well. Enqueue will appear when an entity that su
 ![image](../assets/screenshots/service-call/play7.png)
 
 !!! warning "Radio Mode Media Type"
-    You can only use the album, artist or track media_type if enabling Radio Mode
+    You can only use the album, artist, track or playlist media_type if enabling Radio Mode
```

Comment:
Should this also need a comment that radio mode is only available when you are using streaming provider(s) in MA ? It doesnt work with local files

Reply:
Fixed. We had a note against DSTM but not here